### ISSUE TITLE
Colab disconnects for no reason

### ISSUE THREAD

**Issue by SES428** [etherealxx/etherportal-webui-colab#12]
I have a problem again. The Colab disconnects even if I play the Music Player, I checked the internet and everything is good but the Colab isn't. Did Google just fucking nerf the ability to use music players or is this one of those ai Laws that did this? Im confused

**Comment by SES428** [comment]
Another weird thing. It became disconnected after generating image first time after it was launched. I refreshed every time and the problem still persist

**Comment by etherealxx** [comment]
> Another weird thing. It became disconnected after generating image first time after it was launched. I refreshed every time and the problem still persist

Yeah they nerfed the entire free colab user. everyone got this, every colab that use stable-diffusion-webui. just unfortunate.

**Comment by SES428** [comment]
Yeah, that's unfortunate. Well it was nice using your Collab. Not gonna close this issue as everyone should see this. Thank you for your service.

Edit: I hope the community will find another way for us mobile users to use stable diffusion

**Comment by SES428** [comment]
Why use PC from direct installation while having no problems?
Well I use AMD type PCs which are relatively slower than GPU, I don't know if CPUs are slower or faster than that. It generates 512x512 in around 8-15 minutes in average, take note that I just bought it few months ago and I probably won't spend PCs that runs with Nvidia Drivers. While mobile using colab thanks to the ability to use GPUs can do 768x1024 in just few seconds. It's really boring to generate slow images while doing something else. 
Another is I prefer mobiles, they're portable and easier to setup than PCs.

**Comment by excuse-me-just-minding-my-own-bussiness** [comment]
is it really a nerf or just OOM issues?

**Comment by etherealxx** [comment]
> is it really a nerf or just OOM issues?

It is a nerf. a lot of other stable diffusion colab user have the same exact issue.

**Comment by etherealxx** [comment]
@SES428 to give you a quick easy-to-understand summary, basically when using colab, we're borrowing google's PC. that pc has a gpu in it, Tesla T4 (2070 ish performance). However, since a lot of people use stable diffusion on colab recently, colab team (google) is ran out of GPU, they're burning money. so they did this limitations for stable diffusion colab. paid user won't get this disconnect issues.

**Comment by syzku** [comment]
> > is it really a nerf or just OOM issues?
> 
> It is a nerf. a lot of other stable diffusion colab user have the same exact issue.

So is there any alternatives can be used for this beside of google colab?

**Comment by etherealxx** [comment]
> So is there any alternatives can be used for this beside of google colab?

There are paperspace, kaggle, runpod, etc but i have little knowledge about them. Btw i have another colab that i'm maintaining currently that supposedly can bypass the colab nerf. Try it here: https://github.com/etherealxx/volatile-concentration-localux-colab

**Comment by syzku** [comment]
appreciate it
